app/src/main/java/com/example/citassalon/data/models/room/StaffDao.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.citassalon.data.models.room
+
+import androidx.lifecycle.LiveData
+import androidx.room.*
+import com.example.citassalon.data.models.Staff
+import kotlinx.coroutines.flow.Flow
+
+@Dao
+interface StaffDao {
+
+    @Insert
+    suspend fun insertStaff(staff: StaffName)
+
+    @Insert
+    suspend fun insertAll(vehicle: List<StaffName>)
```

Comment:
Esta parte esta bien nomas que seria recomendable cambiar el nombre de la variable vehicle to staff solo para que se mas compredible